lib/shoryuken/manager.rb

```diff
@@ -3,7 +3,7 @@ class Manager
     include Util
 
     BATCH_LIMIT        = 10
-    HEARTBEAT_INTERVAL = 0.1
+    HEARTBEAT_INTERVAL = 1
```

Comment:
> Although instead of ready.zero?, is there just some type of latch on the pool to await a free thread so you don't have to do the sleep?

@waynerobinson not I'm aware of 🐼